pkg/template/generate.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -e
+
+# This script generates the code for the template package.
+# It takes the code from the Go standard library and applies the patches.
+
+if ! command -v gopatch &> /dev/null; then
+    echo "gopatch could not be found, installing it..."
+    go install github.com/uber-go/gopatch@latest
+fi
```

Comment:
Should we move this to the `internal/tools/tools.go` file? That way when calling `inv install-tools` hey get install for everyone 😄

Reply:
That's a good idea yes !

Reply:
Right now this is not used in CI, but I'd like to make this automated when we do a Go update, so that would be better